### ISSUE TITLE
Report correctly on managed-dependencies

### ISSUE THREAD

**Issue by ikitommi** [xsc/lein-ancient#83]
When versions are defined via `:managed-dependencies`, the report doesn't shot the version. eg..

```clj
(defproject kikka "0.1.0-SNAPSHOT"
  :managed-dependencies [[metosin/spec-tools "0.4.0"]]
  :dependencies [[metosin/spec-tools]])
```

with `lein ancient` it prints:

```clj
[metosin/spec-tools "0.5.0"] is available but we use ""
```

and thanks for the awesome plugin!

**Comment by danieltdt** [comment]
Any news about it?

**Comment by awb99** [comment]
yeah!! A fix for this would really help!

**Comment by awb99** [comment]
Is there anyone stil workinng on lein-ancient?
A fix for managed-dependenies to me is a pretty important feature. I know it is a new leiningen feature, so implementation will take a while. However, the reason to use lein-ancient is for BIG projects, where it is very difficult to manage dependencies. And such projects will use managed-dependencies sooner or later. THANKS!

**Comment by xsc** [comment]
@awb99 I'm slowly finding time to get back into these projects. Since I have not been actively working on them for a few months, it might still take a bit until I've familiarized myself again.

Give me a bit of time - this doesn't look too hard to fix.

**Comment by xsc** [comment]
About 3 years too late, but I just pushed `[lein-ancient "0.7.0-SNAPSHOT"]` which should incorporate a fix for this. Could one of you reading this check it out? Thanks!

**Comment by dancek** [comment]
@xsc `0.7.0-SNAPSHOT` fixed the issue for the parent project that has `:managed-dependencies`.

In a child project, however, all the dependencies are listed with version `""`. Child project meaning that the project.clj has
```
  :plugins [[lein-parent "0.3.8"]]
  :parent-project {:path    "modules/parent/project.clj"
                   :inherit [:managed-dependencies]}
```

In any case, the current fix is very useful. :+1:

**Comment by dcarley** [comment]
I forgot to report it, but experienced the same 👍

**Comment by xsc** [comment]
I deployed another SNAPSHOT that might fix the issue with parent projects. (It now uses the project map that Leiningen provides to find managed dependencies, instead of collecting them from the file itself.)

**Comment by dancek** [comment]
@xsc this new snapshot version shows `all artifacts are up-to-date.` for a child project even when some of the dependencies are out of date. The parent project has correct version listings.

**Comment by xsc** [comment]
@dancek Right now, the plugin won't check dependencies of the parent project if run in a child project. Is that what you mean or are there actual outdated dependencies in the child project's `project.clj`?

**Comment by dancek** [comment]
@xsc yeah, that's what I meant. I have all the dependency versions in the parent project. I.e. I use the parent project mostly to keep multiple child projects using the same dependency versions.

**Comment by dancek** [comment]
In any case, I think the current version is good for release.

**Comment by xsc** [comment]
Current fix released in `[lein-ancient "0.7.0"]`. Tracking the parent dependency topic in #119, please share your opinion there. :)
